### PR TITLE
pls accept dis thx

### PR DESCRIPTION
**Pull request by nickrstewarttds**

### CODE REVIEW

File: dev-thing.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+wheeeeeeeeeeeeeeeeeeee
```
Comment (nikitaqasdet) [inline dev-thing.txt:1]: this is fine

**Review by giefortunaQA — APPROVED** [review]

**Review by nikitaqasdet — APPROVED** [review]
all good

File: dev-thing.txt (hunk 1 above)
Comment (giefortunaQA) [reply dev-thing.txt:1]: whooooooooo

Comment (MuratOzturk18) [inline dev-thing.txt:1]: Don't forget the !!!!!!!

**Review by MuratOzturk18 — APPROVED** [review]

**Pull request merged**